### ISSUE TITLE
Scot - export function

### ISSUE THREAD

**Issue by MeghanUX** [CrowdRescueHQ/CrowdRescue#82]
When dispatch reads off the broadcast, it is often easier to review, edit, and read - from a format which is not a web page. So they export the content to - email, Word, Google (HTML?, other?). It would be nice to have an "export" function for this.

Note: Some people are reading the generated script - off a mobile phone. The web page is not optimized for viewing on that type of screen.

**Comment by ArtOfCode-** [comment]
Optimizing for mobile shouldn't be too difficult. I think we're just missing a `<meta>` tag.

**Comment by izwick-schachter** [comment]
Moved mobile stuff to #87; I've declined unless @ArtOfCode- thinks it's doable without too much trouble.

**Comment by ArtOfCode-** [comment]
Yeah, I'm going to defer it. Adding an "email this to me" button isn't too difficult, we just need to set up SES auth correctly.

**Comment by izwick-schachter** [comment]
I've got no idea how to do this, so if you can do it, do it whenever you have time.
